src/cli/index.ts

```diff
@@ -61,11 +61,7 @@ const optionDefinitions: commandLineArgs.OptionDefinition[] = [
     defaultValue: 'modulizer_out',
     description: 'The directory to write converted files to.'
   },
-  {
-    name: 'in',
-    type: String,
-    description: 'The directory to convert.'
-  },
+  {name: 'in', type: String, description: 'The directory to convert.'},
```

Comment:
Is there a reason you want this arg to be defined on a single line when all the others aren't?

Reply:
It's clang auto-formatting, because it can fit on 1 line it gets auto-formatted. You can see for yourself by running `npm run format`

It can be weird (like here, I wouldn't do this change myself, personally) but it prevents us from having to worry/argue about formatting which is a 💯 win.

Reply:
Alright, that makes sense! :+1: I forgot to run that command on my PR 🙃